Foundation/FileManager.swift

```diff
@@ -1080,7 +1417,10 @@ open class FileManager : NSObject {
         guard alreadyConfirmed || shouldRemoveItemAtPath(path, isURL: isURL) else {
             return
         }
-        
+
+#if os(Windows)
```

Comment:
Want to add an `@availability(Windows, deprecated, message: “Not yet implemented”)` to the method?

Reply:
Oh, wait, I think I get what you mean.